<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ESScope</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView_" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionViewToggleVideo_" />
    <addaction name="actionViewToggleExtractions_" />
    <addaction name="actionViewToggleRangeTruths_" />
    <addaction name="actionViewToggleBugPlots_" />
    <addaction name="actionViewToggleGrid_" />
    <addaction name="actionViewTogglePhantomCursor_" />
    <addaction name="separator" />
    <addaction name="actionAlphaBetaViewFull_" />
    <addaction name="actionAlphaBetaViewPrevious_" />
    <addaction name="actionAlphaBetaViewSwap_" />
    <addaction name="actionAlphaRangeViewFull_" />
    <addaction name="actionAlphaRangeViewPrevious_" />
    <addaction name="actionAlphaRangeViewSwap_" />
    <addaction name="separator" />
    <addaction name="actionViewClearAll_" />
    <addaction name="actionViewClearVideo_" />
    <addaction name="actionViewClearExtractions_" />
    <addaction name="actionViewClearRangeTruths_" />
    <addaction name="actionViewClearBugPlots_" />
    <addaction name="separator" />
    <addaction name="actionPresetSave_" />
    <addaction name="actionPresetRevert_" />
   </widget>
   <addaction name="menuView_" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionViewToggleVideo_" >
   <property name="text" >
    <string>Show Video Data</string>
   </property>
   <property name="shortcut" >
    <string>V</string>
   </property>
  </action>
  <action name="actionViewToggleBinary_" >
   <property name="text" >
    <string>Show Binary Data</string>
   </property>
   <property name="shortcut" >
    <string>B</string>
   </property>
  </action>
  <action name="actionViewToggleExtractions_" >
   <property name="text" >
    <string>Show Extractions</string>
   </property>
   <property name="shortcut" >
    <string>E</string>
   </property>
  </action>
  <action name="actionViewToggleRangeTruths_" >
   <property name="text" >
    <string>Show Range Truths</string>
   </property>
   <property name="shortcut" >
    <string>E</string>
   </property>
  </action>
  <action name="actionViewToggleBugPlots_" >
   <property name="text" >
    <string>Show User Bug Plots</string>
   </property>
   <property name="shortcut" >
    <string>P</string>
   </property>
  </action>
  <action name="actionViewToggleGrid_" >
   <property name="text" >
    <string>Show Grids</string>
   </property>
   <property name="shortcut" >
    <string>G</string>
   </property>
  </action>
  <action name="actionViewTogglePhantomCursor_" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Phantom Cursor</string>
   </property>
   <property name="shortcut" >
    <string>C</string>
   </property>
  </action>
  <action name="actionViewClearAll_" >
   <property name="text" >
    <string>Clear View</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionViewClearVideo_" >
   <property name="text" >
    <string>Clear Video</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionViewClearBinaryVideo_" >
   <property name="text" >
    <string>Clear Binary Video</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionViewClearExtractions_" >
   <property name="text" >
    <string>Clear Extractions</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionViewClearRangeTruths_" >
   <property name="text" >
    <string>Clear Range Truths</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionViewClearBugPlots_" >
   <property name="text" >
    <string>Clear User Bug Plots</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPresetSave_" >
   <property name="text" >
    <string>Save Preset</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPresetRevert_" >
   <property name="text" >
    <string>Revert Preset</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionShowFramesWindow_" >
   <property name="text" >
    <string>Past Frames List</string>
   </property>
   <property name="shortcut" >
    <string>F3</string>
   </property>
  </action>
  <action name="actionShowPlayerWindow_" >
   <property name="text" >
    <string>Past Frames Player</string>
   </property>
   <property name="shortcut" >
    <string>F4</string>
   </property>
  </action>
  <action name="actionAlphaBetaViewFull_" >
   <property name="text" >
    <string>Clear Alpha/Beta View Stack</string>
   </property>
   <property name="toolTip" >
    <string>Clear view stack and show configured view</string>
   </property>
  </action>
  <action name="actionAlphaBetaViewPrevious_" >
   <property name="text" >
    <string>Pop Alpha/Beta View Stack</string>
   </property>
   <property name="toolTip" >
    <string>Restore previous entry in view stack</string>
   </property>
  </action>
  <action name="actionAlphaBetaViewSwap_" >
   <property name="text" >
    <string>Swap Alpha/Beta View Stack Entries</string>
   </property>
   <property name="toolTip" >
    <string>Swap current view stack entry with previous one</string>
   </property>
  </action>
  <action name="actionAlphaRangeViewFull_" >
   <property name="text" >
    <string>Clear Alpha/Range View Stack</string>
   </property>
   <property name="toolTip" >
    <string>Clear view stack and show configured view</string>
   </property>
  </action>
  <action name="actionAlphaRangeViewPrevious_" >
   <property name="text" >
    <string>Pop Alpha/Range View Stack</string>
   </property>
   <property name="toolTip" >
    <string>Restore previous entry in view stack</string>
   </property>
  </action>
  <action name="actionAlphaRangeViewSwap_" >
   <property name="text" >
    <string>Swap Alpha/Range View Stack Entries</string>
   </property>
   <property name="toolTip" >
    <string>Swap current view stack entry with previous one</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
